app/_hub/reedelk/reedelk-transformer/index.md

```diff
@@ -0,0 +1,205 @@
+        request transformation.
+
+    - name: downstream_transformer_url
+      required: no
+      default:
+      value_in_examples: http://myhost/downstream/transform
+      description:
+        The URL of the Reedelk REST flow endpoint to invoke for the Downstream
+        request transformation.
+
+  extra:
+    The `upstream_transformer_url` and `downstream_transformer_url` are the URLs
+    of the Reedelk REST flow endpoint to invoke for the Upstream/Downstream
+    request/response transformations; e.g., `http://localhost:8888/apiabledev/transform`.
+###############################################################################
+# END YAML DATA
+
+###############################################################################
+# BEGIN MARKDOWN CONTENT
+---
+
+## Installation
+
+### Prerequisites
+
+To use the Reedelk plugin for Kong, you must first install the Reedelk IntelliJ
+IDEA flow designer plugin using either of the following options:
+
+- **IntelliJ Marketplace**: From **IntelliJ Preferences > Plugin > Marketplace**,
+  search for `Reedelk`, install the plugin, and restart IntelliJ.
+
+- **Manual install**: From **IntelliJ Preferences > Plugin > Settings icon >
+  Install Plugin From Disk**, and restart IntelliJ.
+
+### Install Kong Plugin Steps
+
+1. Clone the `kong-plugin-reedelk-transformer`:
+
+   ```bash
+      git clone https://github.com/reedelk/kong-plugin-reedelk-transformer.git
+      cd kong-plugin-reedelk-transformer
+   ```
+
+2. Install the module `kong-plugin-reedelk-transformer`:
+
+   ```bash
+   luarocks install kong-plugin-reedelk-transformer-0.1.0-1.all.rock
+   ```
+
+3. Add the custom plugin to `kong.conf`:
+
+   ```
+   plugins = ...,reedelk-transformer
+   ```
+
+4. [Restart](/2.1.x/cli/#kong-restart) Kong.
+
+### Building and Packaging the Kong Plugin
+
+1. The [LuaRocks](http://luarocks.org) package manager must be
+   [installed](https://github.com/luarocks/luarocks/wiki/Download).
+
+2. Build the plugin locally (based on the `.rockspec` in the current directory).
+
+   ```bash
+   $ luarocks make
+   ```
+
+3. Package the plugin.
+
+   ```bash
+   $ luarocks pack kong-plugin-reedelk-transformer 0.1.0-1
+   ```
+
+### Hello World Example
+
+This example uses a pre-packaged `kong-reedelk` docker image with Kong and
+`reedelk-transformer` plugin already installed. You can find the `kong-reedelk`
+docker images on [Dockerhub](https://hub.docker.com/repository/docker/reedelk/kong-reedelk-transformer-plugin).
+
+The `kong-reedelk` image is pre-configured to use the following `kong.yml` file,
+which defines an upstream service mapped on Route `http://localhost:8000/transform`
+and invokes a downstream transformer with the `reedelk-transformer` plugin. The
+configured downstream transformer integration flow URL is:
+
+```
+http://host.docker.internal:8282/api/message
+```
+
+#### Prerequisites
+
+- [Docker](https://www.docker.com/) must be [installed](https://www.docker.com/get-started).
```

Comment:
```suggestion
- Docker must be [installed](https://www.docker.com/get-started).
```
You don't need two links to Docker here. 
Same thing applies to the following prereqs.

Reply:
This is how Reedelk has it on their github doc page. I think it's fine to have both.